app/views/hiring_staff/vacancies/application_details/new.html.haml

```diff
@@ -26,16 +26,12 @@
 
       %div.govuk-form-group#publish_on
         = f.gov_uk_date_field :publish_on,
-                              legend_text: t('jobs.publication_date'),
-                              legend_class: 'govuk-label',
-                              form_hint_text: t('jobs.form_hints.publication_date',
-                                              date: l(Date.today, format: :hinttext))
+                              i18n_attribute: :publish_on,
```

Comment:
I remember you saying that the `i18n_attribute` was optional and that it will use the attribute name if not given.  With the cases where the `i18n_attribute` and the attribute name are the same can we just remove the `i18n_attribute` definition?

Reply:
Thanks for pointing this out. Yes, we can remove them.